techniques/fileDistribution/copyGitFile/2.0/metadata.xml

```diff
@@ -81,7 +81,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           </SECTION>
       <INPUT>
         <NAME>COPYFILE_DESTINATION</NAME>
-        <DESCRIPTION>Destination of the file</DESCRIPTION>
+        <DESCRIPTION>Full path of destination file or directory</DESCRIPTION>
       <LONGDESCRIPTION>This is the absolute path of the file/folder on the managed node</LONGDESCRIPTION>
```

Comment:
i think you can add more details in the long description, like an example
"for instance, /etc/cron.d/0hourly", or whatever loofs relevant